rosie/chamber_of_deputies/tests/test_election_expenses_classifier.py

```diff
@@ -11,16 +11,16 @@ class TestElectionExpensesClassifier(TestCase):
     def setUp(self):
         self.dataset = pd.read_csv('rosie/chamber_of_deputies/tests/fixtures/election_expenses_classifier.csv',
                                    dtype={'name': np.str, 'legal_entity': np.str})
-        self.subject = ElectionExpensesClassifier()
+        self.election_expenser_classifier = ElectionExpensesClassifier()
 
     def test_is_election_company(self):
-        self.assertEqual(self.subject.predict(self.dataset)[0], True)
+        self.assertEqual(self.election_expenser_classifier.predict(self.dataset)[0], True)
 
     def test_is_not_election_company(self):
-        self.assertEqual(self.subject.predict(self.dataset)[1], False)
+        self.assertEqual(self.election_expenser_classifier.predict(self.dataset)[1], False)
 
     def test_fit(self):
```

Comment:
Why is the classifier itself been returned here on fit.

Reply:
I believe that the _fit_ is this case is not supposed to change anything in the data frame (but is kept due to scikitlearn architecture). Can you shed some light on us @irio?